machine/dell/dellemc_s4100_c2338/machine.make

```diff
@@ -0,0 +1,45 @@
+# DELL EMC S4100 Series
+# CPU Module: Intel Atom Rangeley (C2338)
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bcm
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# DELL IANA number
+VENDOR_ID = 674
+
+I2CTOOLS_ENABLE = yes
+I2CTOOLS_SYSEEPROM = no
```

Comment:
`I2CTOOLS_SYSEEPROM = no` is the now the default.  No need to set it.  I would remove this line.